logic/employee.py

```diff
@@ -158,22 +163,15 @@ def _update_employees(employee_dicts):
 
     # Figure out if there are any employees in the DB that aren't in the S3
     # dump. These are terminated employees, and we need to mark them as such.
-    usernames_to_employees = dict(
-        (employee.username, employee)
-        for employee
-        in Employee.query()
-    )
-    db_usernames = set(usernames_to_employees.keys())
+    db_usernames = set(db_employee_dict.keys())
 
     terminated_usernames = db_usernames - current_usernames
     terminated_employees = []
-    for u in terminated_usernames:
-        employee = usernames_to_employees[u]
+    for username in terminated_usernames:
+        employee = db_employee_dict[username]
         employee.terminated = True
         terminated_employees.append(employee)
     ndb.put_multi(terminated_employees)
-    # we need to rebuild the love count index as the departments may have changed.
-    love_count.rebuild_love_count()
```

Comment:
We should update the README to include the call to `rebuild_love_count` in the development section, this might be needed during development. Only calling `load_employees` would not rebuilt the LoveCount. 

See https://github.com/Yelp/love/blame/master/README.md#L100